benchmarks/ephemeris.cpp

```diff
@@ -123,7 +123,8 @@ void BM_EphemerisKSPSystem(benchmark::State& state) {
     astronomy::StabilizeKSP(*at_origin);
     Instant const final_time = at_origin->epoch() + 100 * JulianYear;
     auto const ephemeris = at_origin->MakeEphemeris(
-        FittingTolerance(state.range(0)),
+        /*accuracy_parameters=*/{FittingTolerance(state.range(0)),
```

Comment:
Replace the function `FittingTolerance` by a function `MakeAccuracyParameters` that sets both parameters.

Reply:
As discussed, this doesn't really work well because of the existing `MakeAccuracyParameters` in this file that would still need `FittingTolerance`.